config/application.yml.default

```diff
@@ -190,6 +191,7 @@ lexisnexis_hmac_key_id: pf_iv_hmac_key_id
 lexisnexis_hmac_secret_key: pf_iv_hmac_secret_key
 lexisnexis_instant_verify_timeout: 1.0
 lexisnexis_instant_verify_workflow: gsa2.chk32.test.wf
+lexisnexis_threatmetrix_org_id: test_id
```

Comment:
Interesting, was this configuration value missing previously? Seems like we should have some linting to ensure that all keys in `IdentiyConfig` exist in `application.yml.default`.

Reply:
Yes, Something I noticed is that a lot of the config for threatmetrix doesnt have dummy/default values in application.yml.default